### PR TITLE
Create review join table

### PR DESCRIPTION
**Pull request by b-tramer**

### CODE REVIEW

File: react/src/components/ReviewTile.js [hunk 1]
```diff
@@ -1,16 +1,75 @@
...
+      user_id: this.props.user_id,
+      site_id: this.props.site_id,
+      design_body: this.props.design_body,
+      usability_body: this.props.usability_body,
+      concept_body: this.props.concept_body,
+      vote_count: this.state.vote_count
+    }
+    this.sendVote(votePayload);
+  }
+
```
Comment (nwalberts) [inline react/src/components/ReviewTile.js:30]: You lost me here. Why are you sending ALL of this information in the patch. The only thing you are updating is the vote_count?

File: react/src/components/ReviewTile.js [hunk 2]
```diff
@@ -1,16 +1,75 @@
...
```
Comment (nwalberts) [inline react/src/components/ReviewTile.js:47]: same here

File: react/src/containers/SiteShowContainer.js [hunk 3]
```diff
@@ -73,6 +164,27 @@ class SiteShowContainer extends Component {
...
+          ratingChange = {this.handleRatingChange}
+          userChange = {this.handleUserIdChange}
+          designChange = {this.handleDesignChange}
+          usabilityChange = {this.handleUsabilityChange}
+          conceptChange = {this.handleConceptChange}
+
+          handleSubmit = {this.handleSubmit}
+        />
+      </div>
+
```
Comment (nwalberts) [inline react/src/containers/SiteShowContainer.js:187]: this seems a little extreme. you would probably be much better off moving much of the state that these methods are changing down to the NewReviewForm container, and making that a stateful component.

File: react/src/components/ReviewTile.js (hunk 1 above)
Comment (b-tramer) [reply react/src/components/ReviewTile.js:30]: We weren't able to get the data to persist when only sending the vote_count, so we sent all of the params, even though the user isn't able to edit them.

File: react/src/containers/SiteShowContainer.js (hunk 3 above)
Comment (b-tramer) [reply react/src/containers/SiteShowContainer.js:187]: I'm not sure what you're referring to here.. Are you talking about the upvote and downvote functions?

**Pull request merged**